docs/PUBLISHING.md

```diff
@@ -75,7 +75,7 @@ Make sure you set the following environment variables:
 Run the following command:
 
 ```sh
-make publish-bintray-debian
+make publish-bintray-debian RELEASE_TYPE=<production|development>
```

Comment:
The idea behind it is that the `<environment variable>=<value> make <target>` form will not work in Windows under something like `cmd.exe`, since environment variables have to be declared with `set`. The way it is documented now works across all OSes, which means we don't have to write different instructions for Windows.